chef/cookbooks/updater/recipes/default.rb

```diff
@@ -17,6 +17,9 @@
 # limitations under the License.
 #
 
+::Chef::Recipe.send(:include, CrowbarPacemaker::MaintenanceModeHelpers)
```

Comment:
Just curious - how is this different from `::Chef::Recipe.include CrowbarPacemaker::MaintenanceModeHelpers` ?